requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+flake8
+pytest
+nose
```

Comment:
Consider dropping nose entirely. It is deprecate. (I can send a PR to adapt the test to be 100% `py.test`. The bolierplate reduction is worth it.)